pkgs/tools/package-management/nix/2/10/default.nix

```diff
@@ -0,0 +1,20 @@
+{ common
```

Comment:
The path of this file should be ``pkgs/tools/package-management/nix/2.10.nix``

Reply:
Done.  See 63a6de83ca1149fdc4a9d8930a311c7874315c2a